doc/tutorial.rst

```diff
@@ -138,39 +138,34 @@ If we run this:
 
   robertk01 Desktop$ pylint simplecaesar.py
   ************* Module simplecaesar
-  simplecaesar.py:16:19: C0326: Exactly one space required around assignment
-              encoded=encoded + letters[x]
-                     ^ (bad-whitespace)
-  simplecaesar.py:1:0: C0111: Missing module docstring (missing-docstring)
+  simplecaesar.py:3:0: W0301: Unnecessary semicolon (unnecessary-semicolon)
+  simplecaesar.py:1:0: C0114: Missing module docstring (missing-module-docstring)
   simplecaesar.py:5:0: C0103: Constant name "shift" doesn't conform to UPPER_CASE naming style (invalid-name)
-  simplecaesar.py:6:0: C0103: Constant name "choice" doesn't conform to UPPER_CASE naming style (invalid-name)
-  simplecaesar.py:7:0: C0103: Constant name "word" doesn't conform to UPPER_CASE naming style (invalid-name)
-  simplecaesar.py:8:0: C0103: Constant name "letters" doesn't conform to UPPER_CASE naming style (invalid-name)
```

Comment:
I do not see those warnings anymore. I think it may be due to your `pylint` settings.

Reply:
Your comment seems to be on lines that i've removed (as I don't see those either).

I've copied the output from running pylint 1.6.0 with no custom settings.

Reply:
Yes it concerns lines that have been removed. I don't understand why because with `pylint`'s master branch i still have the message `invalid-name` that is emitted.

Reply:
When I run it (rebased my branch with latest changes from master), it gives the same result, ie:
```
************* Module simplecaesar1
simplecaesar1.py:3:0: W0301: Unnecessary semicolon (unnecessary-semicolon)
simplecaesar1.py:1:0: C0114: Missing module docstring (missing-module-docstring)
simplecaesar1.py:5:0: C0103: Constant name "shift" doesn't conform to UPPER_CASE naming style (invalid-name)
simplecaesar1.py:9:0: C0103: Constant name "encoded" doesn't conform to UPPER_CASE naming style (invalid-name)
simplecaesar1.py:13:12: C0103: Constant name "encoded" doesn't conform to UPPER_CASE naming style (invalid-name)

------------------------------------------------------------------
Your code has been rated at 7.37/10 (previous run: 7.37/10, +0.00)
```

Note that messages are for variables `shift` and `encoded`, not `choice`, `word` or `letters`.

Reply:
@zeckie ok you are right. Thanks.